### PR TITLE
Add salt state module to manage kibana objects

### PR DESCRIPTION
**Pull request by gthvn1**

### CODE REVIEW

**Comment by thouveng** [comment]
!!! Don't merge: work is in progress !!!

File: _states/kibana_object.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        return ret
+
+    if not kibana_type:
+        ret['result'] = False
+        ret['comment'] = 'Type is not set'
+        return ret
+
+    profile = __salt__['config.option']('kibana')
+
+    url = profile.get('kibana_url', None)
```
Comment (elemoine) [inline _states/kibana_object.py]: You don't need the second parameter (`None`).

Comment (gthvn1) [reply _states/kibana_object.py]: Right it is the default.

**Review by simonpasquier — APPROVED** [review]

**Comment by gthvn1** [comment]
I added the possibility to delete an object. I also manage an update.

**Review by simonpasquier — APPROVED** [review]

**Pull request merged**